docs/project-types.md

```diff
@@ -0,0 +1,57 @@
+# EventStoreDB Project Types: Strategies for transitioning to event sourcing with EventStoreDB
+
+EventStoreDB is a powerful database for building event-sourced systems. Understanding the various project types and architectural approaches is crucial for a successful implementation. This overview outlines three primary EventStoreDB project types.
+
+
+## 1. Greenfield projects
+
+Greenfield projects involve starting from scratch, ideal for implementing modern, event-driven architectures from the ground up.
+
+
+### Characteristics
+
+- **Designed events**: Events are specifically designed for the domain, ensuring data relevance and quality.
+- **Data continuity**: Regular updates maintain the relevancy and quality of the data over time.
+- **Modern relevance**: This aligns with current trends in software architecture, like microservices and data meshes.
```

Comment:
Perhaps something like this would be clearer:
```suggestion
- **Modern relevance**: The project is designed using current trends in software architecture, like microservices and data meshes.
```